test/126_observers_can_call_election_3_test.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Observers can call election"
+
+# Start network with sufficient voters for consensus
+weave_on $HOST1 launch --init-peer-count=2
+weave_on $HOST2 launch --init-peer-count=2 $HOST1
+
+# Add an observer
+weave_on $HOST3 launch --init-peer-count=0 $HOST2
+
+# Wait for network to settle; ensure IPAM is idle
+sleep 5
+assert "weave_on $HOST1 status ipam"
+assert "weave_on $HOST2 status ipam"
+assert "weave_on $HOST3 status ipam"
```

Comment:
subtle. deserves a comment. or perhaps add '""' to make it clear that we are asserting that the output is empty.